app/views/mailers/data_export_mailer/export_ready.html.erb

```diff
@@ -0,0 +1,114 @@
+<table
```

Comment:
**suggestion:** Have you tried adding a style for a `font-family` on the table? It might help the response look the more similar on different platforms. try adding (these are the sans-serif fonts I copied from tailwind):

```css
font-family: ui-sans-serif, system-ui, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
```

Reply:
Added in ead4a873f3f8f3ef5ad5dd4f674b2d0a3c897b7e